reactor-core/src/main/java/reactor/core/publisher/FluxHandleFuseable.java

```diff
@@ -123,8 +123,8 @@ public boolean tryOnNext(T t) {
 					}
 					else {
 						reset();
-						return false;
 					}
+					return false;
```

Comment:
not sure about this one. processing the error and terminating was considered as a consumption of the tryOnNext so I think it was the right thing to do to only `return false` when there's a continue.

Reply:
This was already returning false for the `onError` (no failure handling) case, as does `FluxFilter`, so no change there.  And of course, the continue case should return false, and does.

Reply:
wouldn't it previously fall through to `return true` though?
https://github.com/reactor/reactor-core/blob/master/reactor-core/src/main/java/reactor/core/publisher/FluxHandleFuseable.java#L124

Reply:
Take a look at https://github.com/reactor/reactor-core/blob/master/reactor-core/src/main/java/reactor/core/publisher/FluxFilter.java#L116, this is what I was basing things on.  Maybe this is wrong, but all FluxFilter tryOnNext paths use `false` return in case of error with default stop strategy.

Reply:
FluxMap on the other hand returns `true` in this case.  Wondering if this difference is deliberate or it's an inconsistency :-( Any idea?